test/test/mocks/widgets.js

```diff
@@ -1,3 +1,36 @@
+export const BODY_OK =
+{
+  payload: {
```

Comment:
With makeRequest we only need the payload, not the entire response. Please remove payload, errors and metaData level